resources/src/topicHeader.cpp

```diff
@@ -49,6 +49,7 @@ const char* CONTROL_LOADPARAM="/kraken/control/loadparam_srv";
 
 //sensor topics
 const char* SENSOR_IMU="/kraken/sensor/imu";
+const char* SENSOR_IMU_NEW="/kraken/sensor/imu_new";
```

Comment:
@athityakumar You also need to declare this inside the `topicHeader.h` header file. (Extern this variable, and declare it in the class definition)